schemas/bridging/raiseIntentResultResponse.schema.json

```
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/raiseIntentResultResponse.schema.json",
+  "title": "RaiseIntentResultResponse",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeResponse.schema.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "raiseIntentResultResponse"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "intentResult": {
+              "oneOf": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "context": {
+                      "$ref": "../context/context.schema.json"
+                    }
+                  },
+                  "required": ["context"],
+                  "additionalProperties": false
+                },
+                {
+                  "type": "object",
+                  "properties": {
+                    "channel": {
+                      "$ref": "../api/channel.schema.json"
+                    }
+                  },
+                  "required": ["channel"],
+                  "additionalProperties": false
+                }
+              ]
+            }
+          },
+          "required": ["intentResult"],
+          "additionalProperties": false
+        },
+        "meta": true
```

```suggestion
        "meta": {
          "type": "object",
          "properties": {
            "sources": {
              "type": "array",
              "items": {
                "$ref": "../api/appIdentifier.schema.json"
              }
            }
          }
        }
```

not refining error sources as could be either app or desktop agent (say if app went away or app threw error)